theme/static/style/style.css

```diff
@@ -440,6 +444,34 @@ pre code {
         padding-left: 1.8rem;
         text-align: left;
     }
+
+    /*features page*/
+    .features .w50 {
+        padding-top: 2rem;
+    }
+    .features .col-8 {
+        min-width: 100%;
+        max-width: 100%;
+    }
+    .features .col-3 {
+        min-width: 100%;
+        max-width: 100%;
+    }
+    .features img {
+        min-width: 95%;
+        max-width: 95%;
+        float: none;
+        margin: auto;
+        display: block;
+    }
+    .features li {
+        padding-left: 1.8rem;
+        padding-right: 1.8rem;
+    }
+    .features .col-2 {
+        display: none; 
+        width: 0;
+    }
```

Comment:
Looking at the above block I think most of it needs to go into the media selector for width 1360px so we can display images nicely even in landscape mode.